hack/run-reporting-operator-local.sh

```diff
@@ -42,6 +65,44 @@ else
     echo Skipping Prometheus port-forward
 fi
 
+if [ "$METERING_PRESTO_USE_TLS" == "true" ]; then
+    maxTries=50
+    tries=0
+    echo "Getting reporting-operator presto server TLS secrets"
+    until kubectl -n "$METERING_NAMESPACE" get secrets reporting-operator-presto-server-tls -o json > "$TMPDIR/reporting-operator-presto-server-tls.json"; do
+        if [ "$tries" -gt "$maxTries" ]; then
+            echo "Timed out waiting for secret reporting-operator-presto-server-tls"
+            exit 1
+        fi
+        tries+=1
+        echo 'Waiting for secret reporting-operator-presto-server-tls'
+        sleep 5
+    done
+    echo "Getting reporting-operator presto client TLS secrets"
+    until kubectl -n "$METERING_NAMESPACE" get secrets reporting-operator-presto-client-tls -o json > "$TMPDIR/reporting-operator-presto-client-tls.json"; do
+        if [ "$tries" -gt "$maxTries" ]; then
+            echo "Timed out waiting for secret reporting-operator-presto-client-tls"
+            exit 1
+        fi
+        tries+=1
+        echo 'Waiting for secret reporting-operator-presto-client-tls'
+        sleep 5
+    done
+
+    export REPORTING_OPERATOR_PRESTO_USE_TLS=true
+    export REPORTING_OPERATOR_PRESTO_CA_FILE="$TMPDIR/reporting-operator-presto-server-ca.crt"
+    export REPORTING_OPERATOR_PRESTO_USE_AUTH=true
+    export REPORTING_OPERATOR_PRESTO_CLIENT_CERT_FILE="$TMPDIR/reporting-operator-presto-client-tls.crt"
+    export REPORTING_OPERATOR_PRESTO_CLIENT_KEY_FILE="$TMPDIR/reporting-operator-presto-client-tls.key"
+    export REPORTING_OPERATOR_PRESTO_CLIENT_CA_CERT_FILE="$TMPDIR/reporting-operator-presto-client-ca.crt"
+    export REPORTING_OPERATOR_PRESTO_TLS_INSECURE_SKIP_VERIFY=true
```

Comment:
Do we need the client CA file anymore? We deleted that in the most recent update to the Presto stuff.

Also, where else is `REPORTING_OPERATOR_PRESTO_TLS_INSECURE_SKIP_VERIFY` used?

Reply:
Reporting-operator reads the env vars natively, these map to the flags. And yes, `REPORTING_OPERATOR_PRESTO_CLIENT_CA_CERT_FILE` can be removed